modules/rest/openapi/openapi.yaml

```diff
@@ -410,7 +442,6 @@ components:
     Problem:
       required:
       - code
-      - detail
       - invalidParams
```

Comment:
Do we reallly need to mark `detail` as not requered? This is not the only field that might be not requered here.

Reply:
The only reason for now is IDEA inspection which tells me that this is `Nonnull` but I compare it to `null`.

Reply:
I would recommend rather to disable an inspection then introducing inconsistency to the public API spec.

Reply:
Probably it's worth investigatin how to generate proper spec, because now it treats all fields as required, which is not the case.